django-stubs/contrib/gis/gdal/layer.pyi

```diff
@@ -41,7 +41,7 @@ class Layer(GDALBase):
     spatial_filter: OGRGeometry | None
     def get_fields(self, field_name: str) -> list[Any]: ...
     @overload
-    def get_geoms(self, geos: Literal[True] = ...) -> OGRGeometry: ...
+    def get_geoms(self, geos: Literal[True] = ...) -> GEOSGeometry: ...
     @overload
-    def get_geoms(self, geos: Literal[False]) -> GEOSGeometry: ...
```

Comment:
I think these should be `list[*Geometry]`.

Reply:
Thank you!